index.html

```diff
@@ -1,177 +1,196 @@
+    <div class="wrapper hero flex space-between flex-column">
+      <div class="left-side hero flex space-between flex-column">
+        <h2 class="heading">About
+          Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work
+          and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">Let's contact</a>
+        <ul class="social">
+          <li><a href="#"><img src="images/social/twitter.svg" alt="twitter"></a></li>
+          <li><a href="#"><img src="images/social/linkedin.svg" alt="linkedin"></a></li>
+          <li><a href="#"><img src="images/social/medium.svg" alt="medium"></a></li>
+          <li><a href="#"><img src="images/social/github.svg" alt="github"></a></li>
+          <li><a href="#"><img src="images/social/vector.svg" alt="vector"></a></li>
+        </ul>
+        <button class="bg round-borders cap resume" type="button">Get my resume</button>
+      </div>
+
+      <details class="ability" open>
+        <summary>
+          <h3> Languages</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+
+        <ul class="languages">
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/javascript.svg"
+              alt="javascript"><span>JavaScript</span></li>
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/html.svg"
+              alt="html"><span>html</span></li>
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/css.svg" alt="css"> <span>css</span>
+          </li>
+        </ul>
+      </details>
+
+      <details class="abitity">
+        <summary>
+          <h3> Frameworks</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+      </details>
+
+      <details class="abitity">
+        <summary>
+          <h3>Skills</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+      </details>
+    </div>
+  </section>
+  <footer>
+    <div class="footer">
+      <h2 class="heading bold cap">Contact me</h2>
+      <p>If you have an application you are interested in developing,
+        a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+      <form class="form" action="https://formspree.io/f/mlezraod" method="POST">
+        <input class="name" type="text" name="name" placeholder="Reem" maxlength="30" required>
+        <input class="email" type="email" name="email" placeholder="janinareem@gmail.com" required>
+        <textarea class="text-area" name="Write your message here" cols="30" rows="10" maxlength="500"
+          placeholder="Write your message here" required></textarea>
+        <button class="button bg round-borders cap " type="submit">Get It Touch</button>
+      </form>
+    </div>
+  </footer>

```

Comment:
- [ ] (Optional) I think it would be nice to follow the design of Figma and remove the `padding-bottom` or `margin-bottom` used at the bottom of the design. It would be nice to also position the` background-image` around the `button` further right than what we have right now.

## Your design

![image](https://user-images.githubusercontent.com/77106962/144293981-c532044c-bbba-40e0-87c1-2b0c720f3af8.png)


## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144294088-17244970-0896-4541-9f6c-1ce933b170be.png)